doc/source/tune/tutorials/tune-resources.rst

```diff
@@ -18,6 +18,10 @@ of CPUs (cores) on your machine.
     )
     results = tuner.fit()
 
+.. warning::
+
+    ``tune.with_resources`` cannot be used with :ref:`Ray AIR Trainers <air-trainers>`. If you are passing a Trainer to a Tuner, specify the resource requirements in the Trainer instance using :class:`~ray.air.config.ScalingConfig`. The general principles outlined below still apply.
+
 You can override this per trial resources with ``tune.with_resources``. Here you can
```

Comment:
One more suggestion: we don't actually link to `tune.with_resources` anywhere in this guide:

```suggestion
You can override this per trial resources with :ref:`tune-with-resources`. Here you can
```